src/seedu/addressbook/data/person/AddressBlock.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+public class AddressBlock {
+	
+	private final String blockNum;
+	
+	public AddressBlock(String str){
+		this.blockNum = str;	
+	}
+	
+	public String getValue(){
+		return this.blockNum;
+	}
+}
```

Comment:
AddressBlock class (and all other 3 new classes) are a little bare bone :-).

Can consider adding toString(), validation etc.